frontend/src/__tests__/cypress/cypress/tests/e2e/dataScienceProjects/testLaunchStandaloneNotebook.cy.ts

```diff
@@ -0,0 +1,69 @@
+import { HTPASSWD_CLUSTER_ADMIN_USER } from '~/__tests__/cypress/cypress/utils/e2eUsers';
+import { projectListPage } from '~/__tests__/cypress/cypress/pages/projects';
+import { notebookServer } from '~/__tests__/cypress/cypress/pages/notebookServer';
+import {
+  waitForPodReady,
+  deleteNotebook,
+} from '~/__tests__/cypress/cypress/utils/oc_commands/baseCommands';
+
+describe('Verify a Jupyter Notebook can be launched directly from the Data Science Project List View', () => {
+  before(() => {
+    // Check if a notebook is running and delete if it is
+    deleteNotebook('jupyter-nb');
+  });
+
+  it('Verify User Can Access Jupyter Launcher From DS Project Page', () => {
+    // Authentication and navigation
+    cy.step('Log into the application');
+    cy.visitWithLogin('/', HTPASSWD_CLUSTER_ADMIN_USER);
+    projectListPage.navigate();
+
+    // Verify Launch standalone server is available
+    cy.step('Launch Standalone notebook server');
+    projectListPage.findLaunchStandaloneWorkbenchButton().click();
+
+    // Select a notebook image
+    cy.step('Choose Code Server Image');
+    notebookServer.findNotebookImage('code-server-notebook').click();
+
+    // Select the versions dropdown
+    cy.step('Select the code server versions dropdown');
+    notebookServer.findVersionsDropdown('code-server-notebook:2024.1').click();
+
+    // Select a image version
+    cy.step('Select an image version');
```

Comment:
We should specify in the step also the image that is being chosen, it will make things more easier to debug if it fails.

Reply:
updated